models/staging/ecommerce/stg_customers.sql

```diff
@@ -0,0 +1,9 @@
+with customers as (
+
+    select
+        *
```

Comment:
staging models should always list all the columns. This is useful for searching and also to just set the original table rather than just selecting everything

Reply:
makes sense, yml files updated thanks!